scripts/js/lib/api/normalizeResultUrls.ts

```diff
@@ -16,7 +16,7 @@ import { HtmlToMdResultWithUrl } from "./HtmlToMdResult.js";
 import { removePart } from "../stringUtils.js";
 
 export function kebabCaseAndShortenPage(page: string, pkgName: string): string {
-  const kebab = kebabCase(page);
+  const kebab = kebabCase(page).replace(/-v-([0-9]+)/g, `-v$1`);
```

Comment:
Nit to maybe add a comment:

// We also replace values like `-v-2` with `-v2`.